docs/get-started/overview.rst

```diff
@@ -124,3 +124,26 @@ or just cast to a numeric type (e.g. float).::
    # 22530.8
    print(float(unithelper.miles(activity.distance)))
    # 13.9999900581
+
+
+Rate Limits
+===========
+
+Strava imposes rate limits on the usage of its API. This means that the number of
```

Comment:
THANK YOU for updating the docs!!
i'm trying to wrap my head around this
Strava says this - https://developers.strava.com/docs/rate-limits/ : 

Strava API usage is limited on a per-application basis using both a 15-minute and daily request limit. The default rate limit allows 100 requests every 15 minutes, with up to 1,000 requests per day. As an application grows, its rate limit may need to be [adjusted](https://developers.strava.com/docs/rate-limits/#adjustment).

is your language that it's not fixed related to a large use app may be provided with more bandwidth / a higher rate limit? If so can we use that language or link to the docs for more specifics? i'll make a suggestion if you think it's reasonable